movie-app/src/body/Searched.js

```diff
@@ -1,4 +1,4 @@
-import MovieCard from '../components/MovieCard'
+import MovieCard from '../Components/MovieCard'
```

Comment:
The previous naming scheme for the `components` folder is more consistent with React standards.  Generally, the only thing that should start with an upper case letter is an actual component.